src/ResourceManager/Profile/Commands.Profile/packages.config

```diff
@@ -35,8 +35,8 @@
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net452" />
   <package id="Microsoft.PowerShell.5.ReferenceAssemblies" version="1.1.0" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.15" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.14" targetFramework="net452" />
```

Comment:
@ziwa-msft please revert the change made in this file -- we have a separate PR open to update the version of ClientRuntime used throughout PowerShell